apps/teams-test-app/index_cdn.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Teams Test App created using React"
+    />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="./public/manifest.json" />
+    <title>Teams Test App</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <script src="https://res.cdn.office.net/teams-js/2.0.0-beta.1/js/MicrosoftTeams.min.js"
```

Comment:
What are we trying to achieve here? Is it meant to, at some point, validate that a change suggested as a PR doesn't break the script tag based usage of the SDK? If so, I think this is a good start, but it means if I make a change which breaks this set of tests, the PR won't really fail, because the tests will still use the latest stable version of MicrosoftTeams.min.js from the CDN, and not include the changes we're validating.

Reply:
Couple of scenarios I think of the intent of this tests @Ella-ly can confirm:
- Running this test daily\weekly against PROD released version, gives the confidence about our content availability(I hope this PR is intend to that).
- Using this infra setup, we should extend to running tests post every releases to SDF\Prod CDN.

Reply:
For the first bullet, we should have a monitoring setup that consistently checks the availability and raises an IcM ticket if the thing is down. Something like Geneva Monitor (or whatever the new thing is). It should just care about the file being accessible, shouldn't it?

Reply:
I'm trying to take a step back because I'm not sure exactly how this particular file will be used for long-term validation. In my opinion there are at least two things we want to validate:
1. We want to validate that importing the results of the build (MicrosoftTeams.min.js) via HTML script tag works successfully. This should be the case where a change in a PR prevents importing via HTML script tag.
2. We want to validate that importing the package published to prod CDN via HTML script tag works. Having this run on every build seems a little like overkill, but not harmful. This would have to be updated after each release (maybe part of the release build?) is published to prod CDN.

I'm not sure (yet) how this file is being used to achieve either of these scenarios.